h2o-extensions/target-encoder/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -30,13 +31,14 @@
   
     @Override
     public String[] fields() {
-      final List<String> params = extractDeclaredApiParameters(getClass());
+      final List<String> params = new ArrayList<>();
       params.add("model_id");
-      params.add("ignored_columns");
       params.add("training_frame");
       params.add("fold_column");
       params.add("response_column");
-  
+      params.add("ignored_columns");
+      params.addAll(extractDeclaredApiParameters(getClass()));
+
```

Comment:
so the goal is to keep "standard" params in their usual position for the TE widget in Flow?
If so, definitely good, I remember being confused by params last time I saw it: looked like the params were in  random order.

Reply:
maybe you can tweak the `level` in the API annotation as well to emphasize important params, and group others as secondary

Reply:
Great idea. I do think this PR is ready to be merged. Small enough change with a purpose. We might change this everywhere in another PR - it could be nice introduce an order in general. Why not expand the annotation interfaces with ordinals ? Also, we could introduce implicit ordering (depend on class where the parameter was defined - attributes from parent classes go first). Etc etc.

Reply:
@Pscheidl 
> Why not expand the annotation interfaces with ordinals ?
Also thought about it in the past, could be useful in some cases. But as you point out, logic can become tricky when there is inheritance, for example some subclass may want to add a frame and insert it right after`validation frame` (defined in parent class), and I guess it's  difficult to keep this simple without overriding the property to replace the annotation, but then you may get issues if the property is duplicated...
There's room for improvement though.

Reply:
Agree with @sebhrusen that is could be tricky and we might end up with overriding some fields in some cases. As I put it in the corresponding jira "Probably it is done to have flexibility of specifying order on per algo basis just by hardcoding it in the list of fields.".  Unfortunately for developers, hardcoding is sometimes the best/cheap way to provide flexibility/usability for users. It would be great to improve this aspect one day.
I'm not sure if we have a specific idea/proposal and how to formalise it in a separate JIRA but created a JIRA for a reference in the future https://0xdata.atlassian.net/browse/PUBDEV-7375

Reply:
@sebhrusen 
>maybe you can tweak the level in the API annotation for TE as well to emphasize important params, and group others as secondary

thanks, did some changes in levels of the fields